index.html

```diff
@@ -211,6 +219,73 @@ <h2>About me</h2>
           </div>
         </section>
       </div>
+
+      <section class="contact-section">
+        <div>
+          <h2>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+        </div>
+        <div class="contact-form">
+          <form action="https://formspree.io/f/mjvlodlr" method="POST">
+            <ul>
+              <li>
+                <input
+                  type="text"
+                  name="name"
+                  placeholder="Full name"
+                  maxlength="30"
+                  required
+                />
+              </li>
+              <li>
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email address"
+                  required
+                />
+              </li>
+              <li>
+                <textarea
+                  placeholder="Enter text here"
+                  name="message"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </li>
+              <li>
+                <button type="submit" class="button">Get in touch</button>
+              </li>
+            </ul>
+          </form>
+        </div>
+        <div class="icon-wrapper">
+          <ul class="icons">
+            <li>
+              <a href="#"><img src="images/cat.png" alt="cat icon" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/in.png" alt="linkldn icon" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/v.png" alt="two icon" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="images/twitter.png" alt="twitter icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/m.png" alt="m icon" /></a>
+            </li>
+          </ul>
```

Comment:
[OPTIONAL] Kindly increase the gap between the social icons to match the Figma design.
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/159537571-f5158ada-6a87-46f9-b955-7e47da6afc38.png) | ![image](https://user-images.githubusercontent.com/24703171/159537640-64d70e07-8b48-4954-8598-9a2d03d26825.png) |

Reply:
resolved